demo/gallery/bootstrap.js

```diff
@@ -82,56 +82,63 @@ function build(E, log) {
         // does nothing in this test
       }
 
+      async function makeStartingObjs() {
+        const host = await E(vats.host).makeHost();
+        const aliceMaker = await E(vats.alice).makeAliceMaker(host);
+        const bobMaker = await E(vats.bob).makeBobMaker(host);
+        const gallery = await makeGallery(
+          E,
+          log,
+          host,
+          stateChangeHandler,
+          canvasSize,
+        );
+        return {
+          host,
+          aliceMaker,
+          bobMaker,
+          gallery,
+        };
+      }
+
       switch (argv[0]) {
         case 'tapFaucet': {
           log('starting tapFaucet');
-          const aliceMaker = await E(vats.alice).makeAliceMaker();
-          const gallery = makeGallery(E, log, stateChangeHandler, canvasSize);
           log('alice is made');
```

Comment:
no longer relevant (and a few other times below.)

Reply:
Yeah, I had originally taken this out, but then I had to change the tests to match :D